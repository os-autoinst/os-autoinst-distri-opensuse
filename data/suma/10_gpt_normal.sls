partitioning:
    disk1:
        type: DISK
        device: /dev/sda
        disklabel: gpt
        partitions:
            p1:
                 size_MiB: 10
                 flags: bios_grub
            p2:
                 size_MiB: 2000
                 flags: swap
                 format: swap
            p3:
                 flags: 
                 mountpoint: /
                 image: JeOS
            p4:
                 size_MiB: 2000
                 flags: 
                 format: btrfs
                 mountpoint: /data
            p5:
                 size_MiB: 2000
                 flags: swap
                 format: ext4
                 mountpoint: /data2

