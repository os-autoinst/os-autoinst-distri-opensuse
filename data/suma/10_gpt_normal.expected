NAME          MAJ:MIN RM   SIZE RO TYPE  MOUNTPOINT
sda             8:0    0    20G  0 disk  
├─sda1          8:1    0    10M  0 part  
├─sda2          8:2    0     2G  0 part  [SWAP]
├─sda3          8:3    0  14.1G  0 part  /
├─sda4          8:4    0     2G  0 part  /data
└─sda5          8:5    0     2G  0 part  /data2
